launch/rosie_teach_repeat.launch

```diff
@@ -25,6 +25,8 @@
     <remap from="/save_waypoint" to="/arm/save_waypoint"/>
 
     <remap from="/gripper_strength" to="/gripper/gripper_strength"/>
+
+    <param name="family" value="Rosie"/>
```

Comment:
I think we need to set this family parameter for the `base`, `arm`, and `gripper` nodes above, right?

Reply:
arm/families and gripper/family are set in the rosparam block, but the base family name does need to be set now that we changed the default from Rosie